embarkConfig/contracts.js

```diff
@@ -221,8 +228,8 @@ module.exports = {
         args: ["$MetadataStoreProxy", "$EscrowProxy", "$SNT"],
         deps: ['RelayHub'],
         onDeploy: [
-          "EscrowRelay.methods.setRelayHubAddress('$RelayHub').send()",
-          "RelayHub.methods.depositFor('$EscrowRelay').send({value: 10000000000000000})"
+          "EscrowRelay.methods.setRelayHubAddress('$RelayHub').send({gasPrice: " + GAS_PRICE + "})",
+          "RelayHub.methods.depositFor('$EscrowRelay').send({value: 10000000000000000, gasPrice: " + GAS_PRICE + "})"
```

Comment:
Better to use string templates:
```
`EscrowRelay.methods.setRelayHubAddress('$RelayHub').send({gasPrice: ${GAS_PRICE}})`,
```

Reply:
I created a task on pivotal to refactor this data script because it's too ugly now.